R/impact_central.R

```diff
@@ -24,27 +24,32 @@ get_raw_impact_details <- function(con, meta1, burden_outcome, is_under5 = FALSE
   v <- determine_vaccine_delivery(meta1)
   i <- unique(grepl("routine", v))
   j <- unique(grepl("campaign", v))
-  if(i & j & meta1$method[1] == "method2a"){
+  if(any(i) & any(j) & meta1$method[1] == "method2a"){
```

Comment:
Generally R docs suggest preferring long form of boolean operators i.e. `&&` and `||` over `&` and `|`.  From docs

> The longer form is appropriate for programming control-flow and typically preferred in if clauses.

Reply:
Cool. This is done.